src/test/kotlin/no/nav/syfo/dialogmote/api/v2/TildelDialogmoteApiV2Spek.kt

```diff
@@ -0,0 +1,266 @@
+                                addHeader(HttpHeaders.ContentType, ContentType.Application.Json.toString())
+                                setBody(objectMapper.writeValueAsString(newDialogmoteDTO))
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+                        }
+                        with(
+                            handleRequest(HttpMethod.Post, urlMote) {
+                                addHeader(HttpHeaders.Authorization, bearerHeader(veilederCallerToken))
+                                addHeader(HttpHeaders.ContentType, ContentType.Application.Json.toString())
+                                setBody(objectMapper.writeValueAsString(newDialogmoteDTOAnnenArbeidstaker))
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+                        }
+
+                        with(
+                            handleRequest(HttpMethod.Get, urlMoterEnhet) {
+                                addHeader(HttpHeaders.Authorization, bearerHeader(veilederCallerToken))
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+                            val dialogmoteList = objectMapper.readValue<List<DialogmoteDTO>>(response.content!!)
+
+                            dialogmoteList.size shouldBeEqualTo 2
+                            dialogmoteList.any { dialogmoteDTO -> dialogmoteDTO.tildeltVeilederIdent == veilederCallerIdent } shouldBeEqualTo true
+                            dialogmoteList.any { dialogmoteDTO -> dialogmoteDTO.tildeltVeilederIdent == veilederIdentTildelesMoter } shouldBeEqualTo false
+
+                            createdDialogmoterUuids.addAll(dialogmoteList.map { UUID.fromString(it.uuid) })
+                        }
+
+                        with(
+                            handleRequest(HttpMethod.Post, urlTildelMote) {
+                                addHeader(HttpHeaders.Authorization, bearerHeader(veilederCallerToken))
+                                addHeader(HttpHeaders.ContentType, ContentType.Application.Json.toString())
+                                setBody(
+                                    objectMapper.writeValueAsString(
+                                        TildelDialogmoterDTO(
+                                            veilederIdent = veilederIdentTildelesMoter,
+                                            dialogmoteUuids = createdDialogmoterUuids
+                                        )
+                                    )
+                                )
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+                        }
```

Comment:
Dette er jo egentlig det eneste som testes, da er det kanskje dumt å gjøre fire andre api-kall 🤔 Det er bedre å bare legge inn i databasen med en (test)metode og så sjekke at det ser riktig ut etter dette kallet. Da slipper man å være avhengig av at mange andre greier i appen må fungere for at denne testen skal gi riktig resultat 🤷‍♂️